### DISCUSSION TITLE
Board on Missioncontrol

### DISCUSSION THREAD

**Discussion by Marcusus57** [ndeadly/MissionControl#564] [General]
Hello,
It would be fun to make the Tony Hawk board switch compatible for racing games like Mario Kart or boat racing, is it a bad idea for missioncontrol?

**Comment by ndeadly** [comment]
Can you give more info on the controller. Is it bluetooth compatible?

**Reply by Marcusus57** [reply]
The xbox360 version yes.  PS3 and Wii versions have usb dongle

**Reply by ndeadly** [reply]
Xbox 360 wireless is not bluetooth. So all three versions would require a USB adapter. It's probably possible to support it in the future, but currently Mission Control is a bluetooth only project.

**Reply by Marcusus57** [reply]
Ok, yes, there is also the USB adapter for xbox360 controller, actually that makes three.  The board has 4 infrared sensors, but knowing that you have to hold a button for acceleration, then you would have to hide a sensor with a plastic part, or program a side button in press to block mode

**Reply by Marcusus57** [reply]
I admit that it's a bit absurd the idea of ​​the board ^^

**Reply by ndeadly** [reply]
I support the wii balance board, that's probably even sillier ;)
In any case, it will have to wait until I support USB connections.

**Reply by Marcusus57** [reply]
😄 super !

**Reply by Marcusus57** [reply]
For the game "Skate it" on Wii, you can use the Wiiboard + Wiimote + Nunchuk, is this set usable with Missioncontrol?

**Reply by ndeadly** [reply]
Do you mean the Wii Balance Board? All three are supported, but you've got the problem that the Balance Board and the Wiimote are separate controllers with their own bluetooth connection. You could have the console combine them into one by having one pretend to be a left Joycon, and the other be the right one. This is not something currently supported by Mission Control, however.

**Reply by Marcusus57** [reply]
yes the balance board, simulating two joycons is a great idea 😮 do you think it's really doable?

**Reply by ndeadly** [reply]
Yes it's possible to do, but requires a couple of issues to be solved, and an applet/overlay to be written to set it up.
